isobus/src/isobus_virtual_terminal_client.cpp

```diff
@@ -1930,14 +1971,26 @@ namespace isobus
 			set_state(StateMachineState::Disconnected);
 		}
 
-		if ((sendWorkingSetMaintenenace) &&
+		if ((sendWorkingSetMaintenance) &&
 		    (SystemTiming::time_expired_ms(lastWorkingSetMaintenanceTimestamp_ms, WORKING_SET_MAINTENANCE_TIMEOUT_MS)))
 		{
 			txFlags.set_flag(static_cast<std::uint32_t>(TransmitFlags::SendWorkingSetMaintenance));
 		}
+		if ((sendAuxiliaryMaintenance) &&
+		    (SystemTiming::time_expired_ms(lastAuxiliaryMaintenanceTimestamp_ms, AUXILIARY_MAINTENANCE_TIMEOUT_MS)))
+		{
+			/// @todo Only set this flag if the object pool contains a 'Auxiliary Input Type 2 Object' (#65)
```

Comment:
We can actually change this to setting the flag only when `ourAuxiliaryInputs` is not empty I think